package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "mqtt",
   "description": "A library for the MQTT protocol",
-  "version": "2.15.2",
+  "version": "2.15.3",
```

Comment:
Generically, I prefer if PRs did not tweak the version number. This is done as part of the release itself.
I'll land it anyway, but please remember for the next time.

Reply:
Oh, this was weird. Can you please rebase and squash your commits? It seems something went wrong with the merge. and a commit of myself is showing up here.